resources/views/admin/layout/app.blade.php

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    @php
+        $title = 'Danam';
+    @endphp
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>{{ $title }}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="title" content="{{$title}}">
+    <meta name="author" content="ColorlibHQ">
+    <meta name="description" content="AdminLTE is a Free Bootstrap 5 Admin Dashboard, 30 example pages using Vanilla JS.">
+    <meta name="keywords" content="bootstrap 5, bootstrap, bootstrap 5 admin dashboard, bootstrap 5 dashboard, bootstrap 5 charts, bootstrap 5 calendar, bootstrap 5 datepicker, bootstrap 5 tables, bootstrap 5 datatable, vanilla js datatable, colorlibhq, colorlibhq dashboard, colorlibhq admin dashboard">
+    <link rel="stylesheet" href="{{asset('admin/dist/css/overlayscrollbars/overlayscrollbars.min.css')}}" crossorigin="anonymous">
```

Comment:
Suggestion for neat file

You can move all link css to "resources\views\admin\includes\main-css.blade.php" and all js to "resources\views\admin\includes\main-js.blade.php" then write include statement.